src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2301,10 +2301,15 @@ internal enum ErrorCode
         ERR_ParamsCollectionMissingConstructor = 9228,
 
         ERR_NoModifiersOnUsing = 9229,
+        ERR_CannotDynamicInvokeOnRefStruct = 9230,
 
         #endregion
 
+        // Note: you will need to do the following after adding errors:
+        //  1) Update ErrorFacts.IsBuildOnlyDiagnostic (src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs)
```

Comment:
Thanks! :-) 
I've been meaning to add this. I hope I'll notice the comment when adding new codes...